src/hooks/sdk/useSelectAudioInputDevice.tsx

```diff
@@ -6,11 +6,13 @@ import { useCallback } from 'react';
 
 import { useMeetingManager } from '../../providers/MeetingProvider';
 
-export const useSelectAudioInputDevice = () => {
+export const useSelectAudioInputDevice = (): ((
+  device: Device | AudioTransformDevice | null
```

Comment:
In the component specific states I see we have used `undefined` and we take in `null` as well is it due to the fact that JS SDK may return a `undefined` device but can take in `null` device for selection purpose?

What will be the state for the device if we select a `null` device, an empty audio input device?

Reply:
So in my later change. The the value of selected device on mount/unmount will be `undefined`. However in JS SDK the `startAudioInput(device: AudioInputDevice)` supports `null` which stands for the empty/fake audio input device. So have to keep it only for audio input device. This will come with my next PR.